src/sagemaker/pytorch/estimator.py

```diff
@@ -203,6 +204,12 @@ def __init__(
             entry_point, source_dir, hyperparameters, image_uri=image_uri, **kwargs
         )
         if distribution is not None:
```

Comment:
Would this check for all job with distribution rather than modelparallel job only?

Reply:
NVM checked the function. LGTM